test/e2e/broker_trigger_test.go

```diff
@@ -0,0 +1,250 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+
+	"github.com/knative/eventing/test"
+	"github.com/knative/pkg/test/logging"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+)
+
+const (
+	defaultBrokerName            = "default"
+	waitForDefaultBrokerCreation = 5 * time.Second
+	waitForFilterPodRunning      = 30 * time.Second
+	selectorKey                  = "end2end-test-broker-trigger"
+
+	any          = v1alpha1.TriggerAnyFilter
+	eventType1   = "type1"
+	eventType2   = "type2"
+	eventSource1 = "source1"
+	eventSource2 = "source2"
+)
+
+func TestDefaultBrokerWithManyTriggers(t *testing.T) {
```

Comment:
Add a comment saying what this is testing.

Reply:
Done!